ftl_installer/config/Config.py

```diff
@@ -0,0 +1,37 @@
+import yaml
+
+
+class Config(object):
+    """
+    Class to handle the configuration given with `--config` option.
+    """
+
+    def __init__(self):
+        """
+        generate an empty config when no input config file provided.
+        """
+
+        self.config = ""
+
+    def config_from_file(self, config_file):
+        """
+        stores the configuration from file to a config object.
+        """
+
+        with open(config_file) as config_file:
+            self.config = yaml.safe_load(config_file)
+
+    def write_config(self):
+        """
+        writes the config object to a file in block format.
+        """
+
+        with open('config.yml', 'w') as outfile:
```

Comment:
Instead of hard coding config.yml here, the config file to write should be a class attribute. Which makes me think that my initial suggestion of having two different methods for initializing the config object was the wrong approach (sorry, it took me seeing the implementation to see the error).

Instead, what should probably be done instead is to rename the config_from_file method to __init__, and do a check to see if the file already exists. If the file already exists, read in the contents, otherwise create a new empty config.